### PR TITLE
[AAE-5021] Add listPeople method to PeopleContentService

### PR DESCRIPTION
**Pull request by TomikaArome**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/mock/ecm-user.service.mock.ts [hunk 1]
```diff
@@ -49,6 +49,17 @@ export let fakeEcmUser = {
     emailNotificationsEnabled: true
 };
 
+export let fakeEcmUser2 = {
```
Comment (DenysVuika) [inline lib/core/mock/ecm-user.service.mock.ts:52]: use `const` over `let`
```suggestion
export const fakeEcmUser2 = {
```

File: lib/core/mock/ecm-user.service.mock.ts [hunk 2]
```diff
@@ -49,6 +49,17 @@ export let fakeEcmUser = {
...
+    firstName: 'another-fake-first-name',
+    lastName: 'another',
+    displayName: 'admin.adf User',
+    email: 'admin.adf@alfresco.com',
+    company: null,
+    enabled: true,
+    emailNotificationsEnabled: true
+};
+
 export let fakeEcmUserNoImage = {
```
Comment (DenysVuika) [inline lib/core/mock/ecm-user.service.mock.ts:63]:
```suggestion
export const fakeEcmUserNoImage = {
```

File: lib/core/mock/ecm-user.service.mock.ts [hunk 3]
```diff
@@ -91,6 +102,26 @@ export let fakeEcmEditedUser = {
     emailNotificationsEnabled: true
 };
 
+export let fakeEcmUserList = {
```
Comment (DenysVuika) [inline lib/core/mock/ecm-user.service.mock.ts:105]:
```suggestion
export const fakeEcmUserList = {
```

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/permission-manager/components/user-name-column/user-name-column.component.ts [hunk 4]
```diff
@@ -17,8 +17,9 @@
 
 import { Component, Input, OnInit } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
-import { Group, NodeEntry, Person } from '@alfresco/js-api';
+import { Group, NodeEntry } from '@alfresco/js-api';
 import { NodePermissionService } from '../../services/node-permission.service';
+import { EcmUserModel } from '../../../../../../core/models/ecm-user.model';
```
Comment (arditdomi) [inline lib/content-services/src/lib/permission-manager/components/user-name-column/user-name-column.component.ts:22]: Should be @alfresco/adf-core

File: lib/content-services/src/lib/permission-manager/models/member.model.ts [hunk 5]
```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-import { Group, Node, NodeEntry, PermissionElement, Person } from '@alfresco/js-api';
+import { Group, Node, NodeEntry, PermissionElement } from '@alfresco/js-api';
 import { PermissionDisplayModel } from './permission.model';
 import { RoleModel } from './role.model';
+import { EcmUserModel } from '../../../../../core/models/ecm-user.model';
```
Comment (arditdomi) [inline lib/content-services/src/lib/permission-manager/models/member.model.ts:21]: Same

File: lib/content-services/src/lib/permission-manager/services/node-permission.service.ts [hunk 6]
```diff
@@ -16,12 +16,13 @@
...
 
 import { AlfrescoApiService, NodesApiService, SearchService, TranslationService } from '@alfresco/adf-core';
-import { Group, GroupMemberEntry, GroupMemberPaging, Node, PathElement, PermissionElement, Person, QueryBody } from '@alfresco/js-api';
+import { Group, GroupMemberEntry, GroupMemberPaging, Node, PathElement, PermissionElement, QueryBody } from '@alfresco/js-api';
 import { Injectable } from '@angular/core';
 import { forkJoin, from, Observable, of, throwError } from 'rxjs';
 import { map, switchMap } from 'rxjs/operators';
 import { PermissionDisplayModel } from '../models/permission.model';
 import { RoleModel } from '../models/role.model';
+import { EcmUserModel } from '../../../../../core/models/ecm-user.model';
```
Comment (arditdomi) [inline lib/content-services/src/lib/permission-manager/services/node-permission.service.ts:25]: Same

File: lib/process-services-cloud/src/lib/task/task-list/components/service-task-list-cloud.component.spec.ts [hunk 7]
```diff
@@ -23,7 +23,7 @@ import { ServiceTaskListCloudComponent } from './service-task-list-cloud.compone
 import { fakeServiceTask, fakeCustomSchema } from '../mock/fake-task-response.mock';
 import { of } from 'rxjs';
 import { ProcessServiceCloudTestingModule } from '../../../testing/process-service-cloud.testing.module';
-import { Person } from '@alfresco/js-api';
+import { EcmUserModel } from '../../../../../../core/models/ecm-user.model';
```
Comment (arditdomi) [inline lib/process-services-cloud/src/lib/task/task-list/components/service-task-list-cloud.component.spec.ts:26]: Same

File: lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.spec.ts [hunk 8]
```diff
@@ -24,7 +24,7 @@ import { TaskListCloudComponent } from './task-list-cloud.component';
 import { fakeGlobalTask, fakeCustomSchema } from '../mock/fake-task-response.mock';
 import { of } from 'rxjs';
 import { ProcessServiceCloudTestingModule } from '../../../testing/process-service-cloud.testing.module';
-import { Person } from '@alfresco/js-api';
+import { EcmUserModel } from '../../../../../../core/models/ecm-user.model';
```
Comment (arditdomi) [inline lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.spec.ts:27]: Same

**Review by arditdomi — CHANGES_REQUESTED** [review]

**Review by arditdomi — APPROVED** [review]

**Review by sivakumar414ram — APPROVED** [review]

**Pull request merged**